store/migration/10002__1.1.0.sql

```diff
@@ -10,6 +10,7 @@ ALTER TABLE vcs ADD CONSTRAINT vcs_type_check CHECK (type IN ('GITLAB_SELF_HOST'
 ALTER TABLE sheet ADD source TEXT NOT NULL CHECK (source IN ('BYTEBASE', 'GITLAB_SELF_HOST', 'GITHUB_COM')) DEFAULT 'BYTEBASE';
 ALTER TABLE sheet ADD type TEXT NOT NULL CHECK (type IN ('SQL')) DEFAULT 'SQL';
 ALTER TABLE sheet ADD payload JSONB NOT NULL DEFAULT '{}';
+CREATE UNIQUE INDEX idx_sheet_project_id_name_source ON sheet(project_id, name, source);
```

Comment:
We may want source coming before name (project_id, source, name) since source is more like a namespace which should come before the resource.

Reply:
Updated order.